### ISSUE TITLE
Suggestion: Allow a player-controlled kobold to lay an egg on the spot

### ISSUE THREAD

**Issue by Slayerhero90** [naelstrof/KoboldKare#133]
While I understand the utility of non-controlled kobolds having a designated laying device instead of a hotkey or an automatic act, I'd like at least to be able to lay my own kobold's eggs wherever. She gets quite a lot of them, and they get quite large since she has this tendency to run mouth-first into her own cum.

**Comment by naelstrof** [comment]
Yeah I like that idea. I should move the egg routine into the kobold code, then allow other things to trigger it.

**Comment by Godeken** [comment]
@naelstrof keypress but you have to hold the key as long as it normally takes on the animation station - i can try create a movement based animation for it if you want?
